frontend/src/pages/projects/screens/spawner/connections/DetachConnectionModal.tsx

```diff
@@ -17,36 +17,44 @@ export const DetachConnectionModal: React.FC<Props> = ({
   notebookDisplayName,
   onDetach,
   onClose,
-}) => (
-  <Modal isOpen variant="medium" onClose={onClose}>
-    <ModalHeader title="Detach connection?" titleIconVariant={isRunning ? 'warning' : undefined} />
```

Comment:
The `titleIconVariant` is removed in the `ContentModal`, please bring it back to allow the modals to set it.

Reply:
in detachConnectionModal; got it